src/rtt/src/test_serial.cpp

```diff
@@ -0,0 +1,69 @@
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/byte_multi_array.hpp"
+#include "stampede_msgs/msg/dji_packet.hpp"
+
+using namespace std::chrono_literals;
+
+class TestSerialNode : public rclcpp::Node
```

Comment:
Let's also move this file to the dji_serial package, as this is related to dji serial more than rtt or uart.